packages/@webex/plugin-cc/test/unit/spec/HttpRequest.ts

```diff
@@ -0,0 +1,63 @@
+import HttpRequest from '../../../src/HttpRequest';
+import {HTTP_METHODS, WebexSDK} from '../../../src/types';
+
+describe('HttpRequest', () => {
+    let webex: WebexSDK;
+    let httpRequest: HttpRequest;
+
+    beforeEach(() => {
+        webex = {
+            request: jest.fn(),
+        } as unknown as WebexSDK;
+        httpRequest = new HttpRequest(webex);
+    });
+
+    describe('request', () => {
+        const URL = 'https://api.example.com/accessResource';
+        const body = {id: '123', name: 'test'};
+
+        it('should make a GET request successfully', async () => {
+            const mockResponse = {data: 'response data'};
+            (webex.request as jest.Mock).mockResolvedValue(mockResponse);
```

Comment:
We might not require mocking of webex.request just mockresolve and mockreject needs to be done as per our requirement. Check for test references where webex.request is being used.

Reply:
Addressed!